spec/lib/jit_preloader/preloader_spec.rb

```diff
@@ -173,7 +173,11 @@
 
         it "successfully load the rest of association values" do
```

Comment:
I think it would also be worth highlighting that we are expecting to not have a n+1 notification here.
```suggestion
        it "successfully load the rest of association values and does not publish a n+1 notification" do
```